sdk/translation/Azure.AI.Translation.Text/src/Custom/TextTranslationClient.cs

```diff
@@ -186,6 +354,17 @@ public virtual Task<Response<IReadOnlyList<TranslatedTextItem>>> TranslateAsync(
             return this.TranslateAsync(targetLanguages, content.Select(input => new InputTextItem(input)), clientTraceId, sourceLanguage, textType?.ToString(), category, profanityAction?.ToString(), profanityMarker?.ToString(), includeAlignment, includeSentenceLength, suggestedFrom, fromScript, toScript, allowFallback, cancellationToken);
         }
 
+        /// <summary> Translate Text. </summary>
+        /// <param name="options">The client translation options. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"><paramref name="options"/> is null. </exception>
+        public virtual Task<Response<IReadOnlyList<TranslatedTextItem>>> TranslateAsync(TextTranslationTranslateOptions options, CancellationToken cancellationToken = default)
```

Comment:
Now that these methods have been added, the other methods without the options parameter can be removed.